epoch1/cuda/ee_mumu/SubProcesses/P1_Sigma_sm_epem_mupmum/check.cc

```diff
@@ -1,4 +1,5 @@
 #include <algorithm>
+#include <array>
```

Comment:
Why was this needed actually?

Reply:
I don't know who secretly includes `array` in the other setups, but it's certainly not included in `check.cc`.
```
P1_Sigma_sm_epem_mupmum (epoch2_tests)$ make
Makefile:42: No cuda found. Export CUDA_HOME to compile with cuda
clang++  -O3 -std=c++11 -I. -I../../src -I../../../../../tools  -Wall -Wshadow -Wextra -Xclang -fopenmp -lomp -DMGONGPU_COMMONRAND_ONHOST  -c check.cc -o check.o
check.cc:602:27: error: implicit instantiation of undefined template 'std::__1::array<char, 128>'
    std::array<char, 128> nprocbuf;
```

Reply:
I would actually bet on a bug in the standard library, i.e. some header also includes secretly includes array.

Reply:
Thanks. Funny I also tried clang yesterday and I did not see that (I saw a few warnings instead).